app/views/protocols/form/study_form_sections/_interactive_form.html.haml

```diff
@@ -31,16 +31,20 @@
         %label.radio-inline.btn.btn-default#study_selected_for_epic_true_button{ class: protocol.selected_for_epic? ? 'active' : ''}
           = form.radio_button :selected_for_epic, 'true', id: "study_selected_for_epic_true", class: 'hidden'
           = t(:constants)[:yes_select]
-        %label.radio-inline.btn.btn-default#study_selected_for_epic_false_button{ class: protocol.selected_for_epic == false || !USE_EPIC ? 'active': ''}
```

Comment:
This change reverts fixes for the Study Type Questions as part of 2.1.0. I think this branch just needs to be updated with the current v3.0.0.